tests/test_fluent_session.py

```diff
@@ -59,10 +59,10 @@ def print_transcript(transcript):
 
 def test_server_exits_when_session_goes_out_of_scope() -> None:
     def f():
-        session = pyfluent.launch_fluent()
-        _fluent_host_pid = session.connection_properties.fluent_host_pid
-        _cortex_host = session.connection_properties.cortex_host
-        _inside_container = session.connection_properties.inside_container
+        session = pyfluent.launch_fluent().connection_properties
+        _fluent_host_pid = session.fluent_host_pid
+        _cortex_host = session.cortex_host
+        _inside_container = session.inside_container
```

Comment:
I think these aren't good changes, the number of lines or characters are not significantly reduced, and the `session` object is no longer a `session` in this case

I would say either revert these changes, or change `session` to a variable name that indicates this refers to `connection_properties`

Reply:
I have made all recommended changes. Thank you for the feedback